app/controllers/users.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+const error = require('../errors'),
+  User = require('../models').User;
+
+exports.userCreate = (req, res, next) => {
+  User.createUser(req.body)
```

Comment:
Return promise here, what happen if i send additional properties in body ?

Reply:
the additional properties are ignored and are not a problem, but in theory would be a bad request should you control that?

Reply:
Not as bad request, but sanitize object before insert to database